lib/number/human.ex

```diff
@@ -70,6 +70,21 @@ defmodule Number.Human do
     end
   end
 
+  @doc """
+  Adds ordinal suffix (st, nd, rd or th) for the number
+  """
+  def number_to_ordinal(number) when is_integer(number) do
+    sfx = ~w(th st nd rd th th th th th th)
+
+    (number |> Integer.to_string()) <> case number |> rem(100) do
```

Comment:
It isn't necessary to use the pipeline operator all the time; my personal rule is to only use it when I'm calling two or more functions in a row.

```elixir
Integer.to_string(number) <> case rem(number, 100) do
```